index.ts

```diff
@@ -214,6 +225,49 @@ export class BigInt extends Uint8Array {
   }
```

Comment:
Would it make sense to add `toBigDecimal()` to `BigInt` that uses the `bigDecimal.fromBigInt(x: BigInt): BigDecimal` that I proposed above?

Reply:
Done.